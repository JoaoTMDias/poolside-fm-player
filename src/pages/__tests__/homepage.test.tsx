import React from "react";
import { shallow } from "enzyme";

import Homepage from "pages/homepage";

describe("<Homepage />", () => {
	it("should render the homepage", () => {
		const component = shallow(<Homepage />);

		expect(component).toMatchSnapshot();
	});
});
