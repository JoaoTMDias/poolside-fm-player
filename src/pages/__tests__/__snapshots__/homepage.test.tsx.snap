// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> should render the homepage 1`] = `ShallowWrapper {}`;
